src/magic_folder/magic_file.py

```diff
@@ -1286,15 +1389,28 @@ def _done_working(self):
 
     _conflicted.upon(
         _conflict_resolution,
-        enter=_uploading,
-        outputs=[_begin_upload],
+        enter=_creating_snapshot,
+        outputs=[_working, _status_upload_queued, _create_local_snapshot],
         collector=_last_one,
     )
     _conflicted.upon(
         _remote_update,
-        enter=_conflicted,
-        outputs=[],  # probably want to .. do something? remember it?
+        enter=_downloading,
+        outputs=[_working, _status_download_queued, _begin_download]
     )
+    # XXX this is trix-y .. the user could be changing the local
+    # contents because they'll ultimately be doing a "resolve" (with
+    # those contents) __OR__ they could be naively simply editing the
+    # file more.
+    #
+    # ...so I think we _do_ want to queue a local-update: an incoming
+    # resolution may happen, and if we have no local change: great,
+    # apply it. But if we _do_ have a local change, we'd want to
+    # .. conflict again?
+    #
+    # need integration test here: have conflict; A changes local file
```

Comment:
Should such a test be added to this PR? (Or captured in a separate ticket?)

Reply:
https://github.com/tahoe-lafs/magic-folder/issues/756